galpy/potential/Potential.py

```diff
@@ -70,7 +70,7 @@ def __init__(self,amp=1.,ro=None,vo=None,amp_units=None):
         OUTPUT:
         HISTORY:
         """
-        Force.__init__(self,amp=amp,ro=ro,vo=vo,amp_units=amp_units)
+        super(Potential, self).__init__(amp=amp,ro=ro,vo=vo,amp_units=amp_units)
```

Comment:
Please don't change this line (nothing to do with this PR).

Reply:
reverted.